packages/react-scripts/scripts/init.js

```diff
@@ -99,7 +94,26 @@ module.exports = function(
     command = 'npm';
     args = ['install', '--save', verbose && '--verbose'].filter(e => e);
   }
-  args.push('react', 'react-dom');
+
+  // Install devDependencies
+  const devDependencies = require('./utils/devDependencies');
```

Comment:
Generation of the `devDependencies` section inside the `package.json` for the template now based on the `./utils/devDependencies.js` content.